File: Unicatólica/Semestre_2024_1/POO - Programação Orientada a Objetos/Turma A/POO2024A/Aula08_02042024/src/revisao3/Exemplo3.java
package revisao3;

public class Exemplo3 {
	public static void main(String[] args) {
		int vetor[] = {8,9,4,5,7,3,10};
		int soma = 0;
		for(int i = 0; i < vetor.length; i++) {
			System.out.println("Vetor["+i+"]: "+vetor[i]);
			soma = soma + vetor[i];
		}
		System.out.println("\nSoma: "+soma);
	}
}

File: Unicatólica/Semestre_2024_2/Estrutura_de_Dados/Turma B/Codigos/aula_25092024/exemplo3.cpp
#include <iostream>
using namespace std;

int main(){
    int vetor[5] = {2,3,5,7,9};
    int soma = 0;
    for(int i = 0; i < 5; i++){
        soma = soma + vetor[i];
    }
    cout << "Somatorio: " << soma << endl;
    return 0;
}